### DISCUSSION TITLE
Empty lines showing up in lspsaga windows

### DISCUSSION THREAD

**Discussion by gonviegas** [nvimdev/lspsaga.nvim#746] [Q&A] [answered]
Hello.
How can I remove the empty lines showing up?

![2023-01-20-152513_538x469_scrot](https://user-images.githubusercontent.com/47750371/213735858-b2cfc617-de83-4d07-83ae-c3925004516c.png)
![2023-01-20-152438_426x196_scrot](https://user-images.githubusercontent.com/47750371/213735865-5b54df97-d413-48cf-8d45-cec6be127b41.png)

<br>

- When using vim.lsp.buf.hover there are no empty lines:

![2023-01-20-155159_357x173_scrot](https://user-images.githubusercontent.com/47750371/213744063-692a2f98-143b-43df-831f-a3eb6ba3ec28.png)
![2023-01-20-155149_491x821_scrot](https://user-images.githubusercontent.com/47750371/213744069-d85af497-cef6-435a-9f32-27fb02ba3c5b.png)

**Comment by glepnir** [comment, marked as answer]
because the builtin use markdown syntax it remove some codeblock character so you can't see them. in lspsaga use treesitter markdown parser. so you see them.

**Reply by gonviegas** [reply]
I erased a character in a empty line and noticed that. It's a well worth tradeoff for sure. 
Thank you.
